pandas/tests/series/methods/test_tz_localize.py

```diff
@@ -41,6 +41,23 @@ def test_series_tz_localize_ambiguous_bool(self):
         result = ser.dt.tz_localize("US/Central", ambiguous=[False])
         tm.assert_series_equal(result, expected1)
 
+    def test_series_tz_localize_matching_index(self):
```

Comment:
can you also add a test for another accessor (e.g. .hours) as i suspect that was also borked ; you can do this in parameterization or in another test if easier

Reply:
ok working on it